packages/sdk-browser/src/models.ts

```diff
@@ -22,5 +22,5 @@ export type EmbeddedAPIKey = {
 
 export interface SigningSession {
```

Comment:
Should we rename this to `ReadWriteSession`? Given `SigningSession` could be interpreted as for signing transactions, whereas this "session" is enabling both reading from + writing to Turnkey

Reply:
implemented.